### PR TITLE
fix(content): run `typos` against content directory

### PR DESCRIPTION
**Pull request by Malet**

### CODE REVIEW

File: content/platform/impl/infra-connector.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ pre = ""
 Infra connector is a module in the reference core platform that allows to create cloud objects using kubernetes resources.
 
 Current implementation for GCP uses the [k8s config connector](https://github.com/GoogleCloudPlatform/k8s-config-connector)
-Installing these will make available a variety of CRD that will allows to create differente GCP resources without the need to write terraform code. For example:
+Installing these will make available a variety of CRD that will allows to create difference GCP resources without the need to write terraform code. For example:
```
Comment (emmersonkelly) [inline content/platform/impl/infra-connector.md:12]: Think this wants to be "different"?

Comment (Malet) [reply content/platform/impl/infra-connector.md]: You're right, updated

**Review by emmersonkelly — APPROVED** [review]
Looks good to me

**Review by tombart — APPROVED** [review]

**Comment by tmcalves** [comment]
This won't really support fork PR, are you able to create a branch PR?

File: content/platform/impl/infra-connector.md (hunk 1 above)
Comment (tmcalves) [reply content/platform/impl/infra-connector.md]: PT spelling, my bad 😄

**Pull request closed without merging**